stories/navbar.js

```diff
@@ -19,6 +19,29 @@ const user = {
   actions: [{ label: "Log out", onClick: action("Logout clicked") }]
 };
 
+const tabs = [
+  {
+    selected: true,
+    title: "Did ",
```

Comment:
- All the `title` should be more relevant like in the screenshot below even if they are funny :)
- onClick should be ```action( `${title}` clicked")```
- You should add an example to cover this case:
![image](https://user-images.githubusercontent.com/47394132/68472467-e86eb580-0220-11ea-8174-bf0945df0ed4.png)